mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -355,6 +368,19 @@ object LinearSVCModel extends MLReadable[LinearSVCModel] {
 }
 
 /**
+ * Abstraction for Linear SVC Training results.
+ * Currently, the training summary ignores the training weights except
+ * for the objective trace.
+ */
+case class LinearSVCTrainingSummary(
```

Comment:
The classes below `LinearSVCTrainingSummary` are private classes, so I think it would better to keep LinearSVCTrainingSummary there (above the private classes)